test/integration/auth/node_test.go

```diff
@@ -412,84 +402,68 @@ func TestNodeAuthorizer(t *testing.T) {
 		}
 	}
 
-	getNode1CSINodeInfo := func(client csiclientset.Interface) func() error {
+	getNode1CSINode := func(client externalclientset.Interface) func() error {
 		return func() error {
-			_, err := client.CsiV1alpha1().CSINodeInfos().Get("node1", metav1.GetOptions{})
+			_, err := client.StorageV1beta1().CSINodes().Get("node1", metav1.GetOptions{})
 			return err
 		}
 	}
-	createNode1CSINodeInfo := func(client csiclientset.Interface) func() error {
+	createNode1CSINode := func(client externalclientset.Interface) func() error {
 		return func() error {
-			nodeInfo := &csiv1alpha1.CSINodeInfo{
+			nodeInfo := &storagev1beta1.CSINode{
 				ObjectMeta: metav1.ObjectMeta{
 					Name: "node1",
 				},
-				Spec: csiv1alpha1.CSINodeInfoSpec{
-					Drivers: []csiv1alpha1.CSIDriverInfoSpec{
+				Spec: storagev1beta1.CSINodeSpec{
+					Drivers: []storagev1beta1.CSINodeDriver{
 						{
-							Name:         "com.example.csi/driver1",
+							Name:         "com.example.csi.driver1",
 							NodeID:       "com.example.csi/node1",
 							TopologyKeys: []string{"com.example.csi/zone"},
 						},
 					},
 				},
-				Status: csiv1alpha1.CSINodeInfoStatus{
-					Drivers: []csiv1alpha1.CSIDriverInfoStatus{
-						{
-							Name:                  "com.example.csi/driver1",
-							Available:             true,
-							VolumePluginMechanism: csiv1alpha1.VolumePluginMechanismInTree,
-						},
-					},
-				},
 			}
-			_, err := client.CsiV1alpha1().CSINodeInfos().Create(nodeInfo)
+			_, err := client.StorageV1beta1().CSINodes().Create(nodeInfo)
 			return err
 		}
 	}
-	updateNode1CSINodeInfo := func(client csiclientset.Interface) func() error {
+	updateNode1CSINode := func(client externalclientset.Interface) func() error {
 		return func() error {
-			nodeInfo, err := client.CsiV1alpha1().CSINodeInfos().Get("node1", metav1.GetOptions{})
+			nodeInfo, err := client.StorageV1beta1().CSINodes().Get("node1", metav1.GetOptions{})
 			if err != nil {
 				return err
 			}
-			nodeInfo.Spec.Drivers = []csiv1alpha1.CSIDriverInfoSpec{
+			nodeInfo.Spec.Drivers = []storagev1beta1.CSINodeDriver{
 				{
-					Name:         "com.example.csi/driver1",
+					Name:         "com.example.csi.driver2",
 					NodeID:       "com.example.csi/node1",
 					TopologyKeys: []string{"com.example.csi/rack"},
 				},
 			}
-			nodeInfo.Status.Drivers = []csiv1alpha1.CSIDriverInfoStatus{
-				{
-					Name:                  "com.example.csi/driver1",
-					Available:             true,
-					VolumePluginMechanism: csiv1alpha1.VolumePluginMechanismInTree,
-				},
-			}
-			_, err = client.CsiV1alpha1().CSINodeInfos().Update(nodeInfo)
+			_, err = client.StorageV1beta1().CSINodes().Update(nodeInfo)
 			return err
 		}
 	}
-	patchNode1CSINodeInfo := func(client csiclientset.Interface) func() error {
+	patchNode1CSINode := func(client externalclientset.Interface) func() error {
```

Comment:
Why do we need to patch the csinode object? Is the driver name that's returned wrong? Should we fix the driver name in the actual driver instead?

Reply:
Sorry I thought this was the e2e test using mock driver. Ignore me.